pages/_app.tsx

```
@@ -34,24 +35,22 @@ export default function App({ Component, pageProps: { session, ...pageProps } }:
+					<title>Vibinex • Review PRs 10x Faster on GitHub</title>
+					<meta name="description" content="Makes it easy to navigate, understand and review code changes in GitHub pull requests by adding context and personalization." />
 					<meta name="keywords" content={keywords.join(", ")} />
-
-					{/* Microsoft Clarity Update */}
-					<script
-						dangerouslySetInnerHTML={{
-							__html: `
+				</Head>
+				{/* Microsoft Clarity Update */}
+				<Script
+					dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)